app/apis/api_kubernetes.py

```diff
@@ -359,6 +361,8 @@ def update_dbs_kubernetes(kubernetes_id:int, applist:set=None, description:str=N
 def create_or_update_drucker_on_kubernetes(
         kubernetes_id:int, args:dict, service_name:str=None):
     app_name = args['app_name']
+    if len(app_name) >= 20:
```

Comment:
Will this raise Exception if len(app_name) is just 20?
">=" should be ">"?

Reply:
Yes you are right. I will fix it. Thank you!